src/plugins/vis_type_timelion/server/series_functions/es/lib/agg_body.js

```diff
@@ -6,21 +6,12 @@
  * Side Public License, v 1.
  */
 
-export function buildAggBody(fieldName, scriptedFields) {
-  const scriptedField = scriptedFields.find((field) => {
-    return field.name === fieldName;
-  });
+export function buildAggBody(fieldName, scriptFields) {
+  const [, script] = Object.entries(scriptFields).find(([name]) => name === fieldName) ?? [];
```

Comment:
doesn't have a lot of sense. It equivalent of 
![image](https://user-images.githubusercontent.com/20072247/116387424-78dcf380-a823-11eb-884c-1b6a67e49ef4.png)